test/test_lexers.jl

```diff
@@ -66,7 +66,6 @@ end
         lexer_test("\"proto3\"", Tokens.STRING_LIT)
         lexer_test("'\uffff'", Tokens.STRING_LIT)
         lexer_test("'\xff'", Tokens.STRING_LIT)
-        lexer_test("'\777'", Tokens.STRING_LIT)
```

Comment:
Oh, well spotted. I think all of the test cases with escape sequences were supposed to be raw strings

Reply:
I'll push my fix to this PR, would that be ok?